.github/workflows/fast-forward.yml

```
@@ -119,13 +123,23 @@ jobs:
       - name: Setup
         uses: ./.github/actions/node-setup
 
+      - name: Is versioning PR
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:4:35: Double quote to prevent globbing and word splitting

(shellcheck)